From 905868b59a760b3df8ce9ae406534a8299e0a0e2 Mon Sep 17 00:00:00 2001
From: "Robert P. J. Day" <rpjday@crashcourse.ca>
Date: Fri, 27 May 2011 16:04:03 -0400
Subject: mmc: Standardize header file inclusion checks.

Standardize the checks for multiple MMC header file inclusion,
including adding comments to terminating #endif's, and fixing
one incorrect comment.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>
Signed-off-by: Chris Ball <cjb@laptop.org>
(cherry picked from commit 100e918610b7487fa18db97b3879cd8d1fdd5974)

Conflicts:

	include/linux/mmc/boot.h
	include/linux/mmc/card.h
	include/linux/mmc/core.h
	include/linux/mmc/dw_mmc.h
	include/linux/mmc/host.h
	include/linux/mmc/ioctl.h
	include/linux/mmc/mmc.h
	include/linux/mmc/pm.h
	include/linux/mmc/sd.h
	include/linux/mmc/sdhci-spear.h
	include/linux/mmc/sdhci.h
	include/linux/mmc/sdio.h
	include/linux/mmc/sdio_func.h
	include/linux/mmc/sdio_ids.h
	include/linux/mmc/sh_mobile_sdhi.h
	include/linux/mmc/tmio.h

Signed-off-by: Simon Horman <horms@verge.net.au>
---
 include/linux/mmc/sh_mmcif.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/linux/mmc/sh_mmcif.h b/include/linux/mmc/sh_mmcif.h
index 9eb9b4b..0222cd8 100644
--- a/include/linux/mmc/sh_mmcif.h
+++ b/include/linux/mmc/sh_mmcif.h
@@ -11,8 +11,8 @@
  *
  */
 
-#ifndef __SH_MMCIF_H__
-#define __SH_MMCIF_H__
+#ifndef LINUX_MMC_SH_MMCIF_H
+#define LINUX_MMC_SH_MMCIF_H
 
 #include <linux/io.h>
 #include <linux/platform_device.h>
@@ -220,4 +220,4 @@ static inline void sh_mmcif_boot_init(void __iomem *base)
 	sh_mmcif_boot_cmd(base, 0x03400040, 0x00010000);
 }
 
-#endif /* __SH_MMCIF_H__ */
+#endif /* LINUX_MMC_SH_MMCIF_H */
-- 
1.7.10.2.565.gbd578b5

